cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -4620,4 +4620,67 @@ mod tests {
         expect_reserved_ident("foo::in::principal", "in");
         expect_reserved_ident("foo::is::bar::principal", "is");
     }
+
+    #[test]
+    fn invalid_single_eq() {
```

Comment:
These might more naturally belong in the `invalid_scope_operator()` test.  At least the ones that have `=` in the scope.  Or some other reorg that brings the single-eq tests there into this test.